### PR TITLE
Plain JavaScript version

### PR DESCRIPTION
**Pull request by JhonatanErosa**
### Awesome books: plain JavaScript with objects

- Add HTML structure
- Add showBooks function
- Add addBooks function
- Add removeBooks function
- Add LICENSE file
- Configure README file
- Fix linters

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    const bookContainer = document.createElement('div');
+    bookContainer.innerHTML = `
+        <p>${book.title}</p>
+        <p>${book.author}</p>
+        <button class='removeButton' data-index='${index}'>Remove</button>
+        <hr/>
+        `;
+    bookList.appendChild(bookContainer);
+  });
+}
```
Comment (Alusp) [inline script.js]: - [ ] Good job so far, :clap: However, the input data can neither be added to the browser Localstorage nor shown on the browser due to the following error shown below, kindly resolve it. :slightly_smiling_face: 
- [ ] Also remember to use arrays in your Js file that holds the title and author.

Error

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @JhonatanErosa and @umarahgul , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close

 ## Highlights

- [ ] Implements basic structures correctly  ✔️
 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello @JhonatanErosa 👋,

_This is an additional review as a result of the second opinion request._

I'm afraid the CR is right, kindly restart your localstorage and re-test the app from your end.

Kindly follow the hint given in the[ project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20collection%20that%20keeps%20a%20list%20of%20books%20(hint%3A%20you%20can%20use%20an%20array%20of%20objects%20for%20that).).

The item that has been added is not displayed on the page and is not saved in the localstorage. 

screenshot:
image

Kindly fix this error and ask for another review.

Keep rocking 🚀!

**Review by usorfaitheloho — APPROVED** [review]
Hi @ , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**